roles/common/tasks/install-upgrade-pypi.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install and upgrade Python packaging and debugging tools.
+  become: true
+  pip:
+    executable: pip3
```

Comment:
Modified the pip config on the prod servers and tested with this feature branch, so using `pip3` is good!